test/caseSpec.js

```diff
@@ -166,15 +166,15 @@ describe('Cases', function () {
       exampleCase.exec_cond_fulfillment = this.exampleFulfillment
       exampleCase.state = 'executed'
 
-      yield this.request()
+      const response = yield this.request()
         .put(exampleCase.id + '/fulfillment')
         .send('cf:0:ZXhlY2V0ZQ')
         .expect(422)
-        .expect({
-          id: 'UnmetConditionError',
-          message: 'Invalid fulfillment: Error: Fulfillment does not match condition'
```

Comment:
Prefer to check the beginning of the string only -- so that if the formatting change again or info are added we don't need to change the tests. In fact I don't even think we should check the string. Checking the status code should be sufficient.